# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1337371747 25200
#      Fri May 18 13:09:07 2012 -0700
# Node ID af953acc055d901ae28c10038aaa06f43e2db12d
# Parent  b9151573a23045aa3ea79ef6e406607ee113a731
build: Enable Autotools 'distcheck' target to complete successfully.

* Makefile.am: Add built file 'ChangeLog' to list of files to delete
during distclean.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -81,17 +81,19 @@ noinst_SCRIPTS = run-octave
 
 INFO_FILES = \
   AUTHORS \
   BUGS \
   INSTALL.OCTAVE
 
 CLEANFILES = $(BUILT_SOURCES)
 
-DISTCLEANFILES = .gdbinit 
+DISTCLEANFILES = \
+  .gdbinit \
+  ChangeLog
 
 MAINTAINERCLEANFILES = $(BUILT_DISTFILES)
 
 CONFIG_FILES = @ac_config_headers@ @ac_config_files@
 
 nodist_octinclude_HEADERS = config.h
 
 all-local: $(noinst_SCRIPTS) $(INFO_FILES) .gdbinit
